.env

```diff
@@ -8,10 +8,10 @@ APP_DESCRIPTION=Visualization, Exploration, and Data Analysis (VEDA)
 APP_CONTACT_EMAIL=email@example.org
 
 # Endpoint for the Tiler server. No trailing slash.
-API_RASTER_ENDPOINT='https://openveda.cloud/api/raster'
+API_RASTER_ENDPOINT='https://staging.openveda.cloud/api/raster'
 
 # Endpoint for the STAC server. No trailing slash.
-API_STAC_ENDPOINT='https://openveda.cloud/api/stac'
+API_STAC_ENDPOINT='https://staging.openveda.cloud/api/stac'
```

Comment:
this is still staging? has the data not been pushed to prod yet? if it is, revert this back to production urls

Reply:
I started a PR a while back for these but I think we need a bit more info to get to production https://github.com/NASA-IMPACT/veda-data/pull/120

Reply:
^ this was actually a guess at a need when I opened it (right after deleting the old collections). Now it sounds like we want to move forward with these

Reply:
This PR can only be merged once all data is available from production data services and we switched the endpoint URLs back to those.

Reply:
is there a status update on this @j08lue and @anayeaye?